### PR TITLE
NYC buildings updates and deployment

### PR DESCRIPTION
**Pull request by jlstevens**
Updates the nyc_buildings projects and adds a deployment.

### CODE REVIEW

**Comment by jlstevens** [comment]
Updated the notebook with the changes from PR #149

File: nyc_buildings/anaconda-project.yml [hunk 1]
```diff
@@ -9,17 +9,17 @@ labels:
 user_fields: [labels, skip, maintainers]
 
 channels:
-- pyviz/label/dev
+  - pyviz
```
Comment (jbednar) [inline nyc_buildings/anaconda-project.yml:12]:
```suggestion
- pyviz
```

**Review by jbednar — APPROVED** [review]
Looks good!

**Comment by jlstevens** [comment]
@jbednar The website has now been updated. Ready to merge...

**Review by jbednar — APPROVED** [review]
Thanks!

**Pull request merged**